js/index.js

```diff
@@ -0,0 +1,192 @@
+
+let middle_img = document.querySelector(".middle-img");
+middle_img.setAttribute("src", "img/mid-page-accent.jpg");
+
+let services_h4 = document.getElementsByTagName("H4")[2];
+services_h4.innerHTML = "Services";
+
+let services_content = document.getElementsByTagName("p")[2];
+services_content.innerHTML = "Services content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.";
+
+let product_h4 = document.getElementsByTagName("H4")[3];
+product_h4.innerHTML = "Product";
+
+let product_content = document.getElementsByTagName("p")[3];
+product_content.innerHTML = "Product content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.";
+
+let vision_h4 = document.getElementsByTagName("H4")[4];
+vision_h4.innerHTML = "Vision";
+
+let vision_content = document.getElementsByTagName("p")[4];
+vision_content.innerHTML = "Vision content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.";
+
+let contact_h4 = document.getElementsByTagName("H4")[5];
+contact_h4.innerHTML = "Contact";
+
+let address = document.getElementsByTagName("p")[5];
+address.innerHTML = "123 Way 456 Street Somewhere, USA";
+
+let phone = document.getElementsByTagName("p")[6];
+phone.innerHTML = "1 (888) 888-8888";
+
+let email = document.getElementsByTagName("p")[7];
+email.innerHTML = "sales@greatidea.io";
+
+let copyright = document.getElementsByTagName("p")[8];
+copyright.innerHTML = "Copyright Great Idea! 2018";
+
+let header = document.getElementsByTagName("header")[0].children;
+let news = document.createElement("a");
+let support = document.createElement("a");
+news.innerHTML = "News";
+support.innerHTML = "Support"
+header[0].prepend(news);
+header[0].appendChild(support);
+
+let nav = document.getElementsByTagName("nav")[0].children;
+let navArr = Array.from(nav);
+
+navArr.forEach((item, index) => {
+  item.style.cssText = 'color: limegreen;'
+})
+
+
+//--------------Event LIsteners---------------------------
+
+
+middle_img.addEventListener('mouseover', (event) => {
+  // Do something!  
+    middle_img.setAttribute("src", "img/matrix.jpg");
+});
+
+middle_img.addEventListener('mouseleave', (event) => {
+  // Do something!  
+    middle_img.setAttribute("src", "img/mid-page-accent.jpg");
+});
+
+
+let heading = document.querySelector('h1');
+
+cta_button.addEventListener('click', (event) => {
+  // Do something!  
+  heading.classList.add("explode");
```

Comment:
I really like the use of additional classes to implement these effects. Excellent work.